lib/Language.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+namespace BeGateway;
+
+class Language
+{
+    public static function getSupportedLanguages()
+    {
+        return [
+          'en', 'es', 'tr', 'de', 'it', 'ru', 'zh', 'fr', 'da', 'fi', 'no', 'pl', 'sv', 'ja', 'be', 'ka', 'uk',
```

Comment:
add lv, az, ro

Reply:
I'll add it now, but they weren't there and they weren't removed, it's not in my competence.